File: src/main/java/tech/zolhungaj/amqapi/servercommands/globalstate/FileServerStatus.java
package tech.zolhungaj.amqapi.servercommands.globalstate;

import com.squareup.moshi.Json;
import tech.zolhungaj.amqapi.servercommands.CommandType;

@CommandType("server state change")
public record FileServerStatus(
        @Json(name = "name")
        String serverName,
        boolean online
){}

File: src/main/java/tech/zolhungaj/amqapi/servercommands/globalstate/FileServerStatus.kt
package tech.zolhungaj.amqapi.servercommands.globalstate

import com.squareup.moshi.Json
import tech.zolhungaj.amqapi.servercommands.CommandType

@JvmRecord
@CommandType("server state change")
data class FileServerStatus(
    @Json(name = "name") val serverName: String,
    val online: Boolean
)
